cinema/serializers.py

```diff
@@ -1,13 +1,71 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
+
+
+class ActorSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    first_name = serializers.CharField(max_length=64)
+    last_name = serializers.CharField(max_length=64)
```

Comment:
What will be if Serializer will allow length 64 to be written to database, and then Model has 55 length allowed?